snet_cli/mpe_service_command.py

```diff
@@ -171,6 +174,40 @@ def print_service_metadata_from_registry(self):
         metadata      = self._get_service_metadata_from_registry()
         self._printout(metadata.get_json_pretty())
 
+    def _service_status(self, url, secure=True):
+        channel = None
+        try:
+            channel = open_grpc_channel(endpoint=url)
+            stub = heartb_pb2_grpc.HealthStub(channel)
+            try:
+                response = stub.Check(heartb_pb2.HealthCheckRequest(service=""), timeout=10)
+            except grpc.RpcError as err:
+                err_code = str(err.code())
+                print(err_code)
+            if response!=None and response.status == 1:
+                return True
+            return False
+        except Exception as e:
+            print("error in making grpc call::url: ", url, "|err: ", e)
+            return False
+
+    def print_service_status(self):
+        metadata      = self._get_service_metadata_from_registry().m
```

Comment:
Please don't use internal structure of service_metadata. Too bad python doesn't have private variables... If you need some functionality from ```MPEServiceMetadata``` which it doesn't have, please add this functionality inside MPEServiceMetadata. 

Which functionality do we want in case of multiply payment groups? I see two options.
1. Force user to obligatory specify ```group_name``` in case of multiply group names, as we do in the rest of the code. 
In this case you simply could write:
```python
        metadata      = self._get_service_metadata_from_registry()
        endpoints = metadata.get_endpoints_for_group(self.args.group_name)
        for endpoint in endpoints:
            status = "Available" if self._service_status(url=endpoint)  else "Not Available"
            self._printout("%s   %s"%(endpoint, status))
```

2.  Print status for all endpoints in all groups. Now I understand that your initial code did something like this. Sorry that I didn't realize it initially. You only should use ```get_all_endpoints``` function which we already have in metadata class. And for course you don't need group_name parameter in this case.
```python
        metadata      = self._get_service_metadata_from_registry()
        endpoints = metadata.get_all_endpoints()
        for endpoint in endpoints:
            status = "Available" if self._service_status(url=endpoint)  else "Not Available"
            self._printout("%s   %s"%(endpoint, status))
```

I think that both solutions are ok...